dockerfiles/vcpkg/core.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:22.04 AS base_build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y
+
+WORKDIR /hpcc-dev
+
+COPY ./*.deb .
+
+RUN apt-get install --no-install-recommends -f -y ./*.deb
+
+EXPOSE 8010
+
+ENTRYPOINT ["touch /var/log/HPCCSystems/myesp/esp.log && \
```

Comment:
question: why is the touch necessary?

Reply:
Whoops that file shouldn't be in this PR...

Reply:
@jakesmith - I removed this file